cmd/joy/release.go

```diff
@@ -222,9 +241,12 @@ func NewReleasePromoteCmd(preRunConfigs PreRunConfigs) *cobra.Command {
 	cmd.Flags().StringSliceVar(&templateVars, "template-var", nil, "Variable to pass to PR/commit templates, in the form of key=value (flag can be specified multiple times)")
 	cmd.Flags().BoolVarP(&narrow, "narrow", "n", false, "Use narrow columns mode")
 	cmd.Flags().BoolVarP(&wide, "wide", "w", false, "Use wide columns mode")
+	cmd.Flags().BoolVar(&all, "all", false, "select all releases")
```

Comment:
I wonder if it would make more sense to pass this as a positional arg like a release, where if `release=all`, then load all.

Reply:
I know some apis do this, but I don't love it. Now your positional args has a special name that releases can't have. 
I know its not a name we would ever use. But still. The dashes to make it a flag is tried and true!

Reply:
Not super in favour of special-meaning names, we successfully were able to avoid those up to now! 😁 `--all` also has the great advantage of being self-documenting, as opposed to special names.

Reply:
Fair enough, we got a check to only accept the positional arg(s) or the flag, so we should be good.

https://github.com/nestoca/joy/pull/151/files?w=1#diff-decfa809d626e940d8f9f17e4be211eb2af9b99e02cbd12d93500b3e5cc73a3eR156